src/xbot/common/subsystems/pose/commands/SetRobotHeadingCommand.java

```diff
@@ -23,6 +23,7 @@ public void setHeadingToApply(double heading) {
     @Override
     public void initialize() {
         log.info("Initializing");
+        log.info("Setting heading to " + heading);
```

Comment:
With a newline character in the middle? I figure it's easier to read as two separate lines than as one concatenated string.